doc/01-authoring-azure-cli-commands.md

```diff
@@ -32,11 +68,37 @@ Minimal file must exist in
       namespace: azure.mgmt.healthcareapis
     ```
 
+Minimal **readme.az.md** file must exist: 
+
+    ## AZ
+
+    These settings apply only when `--cli` is specified on the command line.
+
+    ``` yaml $(az)
+    az:
+      namespace: azure.mgmt.healthcareapis
+    ```
+
+### Python SDK
+
+Python SDK will be automatically generated, however appropriate folder needs to be specified.
+Add following line to **readme.az.md**:
+
+
+    ``` yaml $(az)
+    az:
+      ...
+      python-sdk-output-folder: "$(output-folder)/src/<extension-name>/azext_<extension-name>/vendored_sdks/<sdk-namespace-name>"
+    ```
+
+NOTE: In the future this step may be automated, but for now it's necessary it matches extension folder structure.
+
+
 ### Generate Extension
 
 To generate extension use following command:
 
-    autorest --az --output-folder=/_/azure-cli-extensions /_/azure-rest-api-specs/specification/frontdoor/resource-manager/readme.md
+    autorest-beta --az --output-folder=/_/azure-cli-extensions /_/azure-rest-api-specs/specification/attestation/resource-manager/readme.md
```

Comment:
you probably need to add --version=3.0.6212 here. Because somehow the core will cause some problems

Reply:
and also change "autorest-beta" to autorest?

Reply:
yes